packages/langium/src/lsp/language-server.ts

```diff
@@ -38,6 +37,7 @@ import { mergeCompletionProviderOptions } from './completion/completion-provider
 import { DefaultSemanticTokenOptions } from './semantic-token-provider.js';
 import { mergeSignatureHelpOptions } from './signature-help-provider.js';
 import { URI } from '../utils/uri-util.js';
+import type { LangiumServices, LangiumSharedServices } from './lsp-services.js';
```

Comment:
An invocation of 'Organize imports` would by really nice 😉

Reply:
Good point, I'll add that real quick.